app/src/main/java/zafrani/com/pubgapp/models/Category.java

```diff
@@ -5,22 +5,41 @@
 
 import com.google.gson.annotations.SerializedName;
 
+import java.util.List;
+
 public class Category {
+    @NonNull
     @SerializedName("cat_name")
-    private String name;
+    private final String name;
+
+    @NonNull
+    @SerializedName("types")
+    private final List<Type> types;
+
+
+    public Category(@NonNull final String cat_name,
+                    @NonNull final List<Type> types
+                    ) {
+        this.name = cat_name;
+        this.types = types;
+    }
 
-    public Category(@NonNull final String name) {
-        this.name = name;
+    @NonNull
+    public List<Type> getTypes() {
+        return types;
     }
 
+    @NonNull
     public String getName() {
         return name;
     }
 
+    @NonNull
     @Override
     public String toString() {
         return "Category{" +
-               "name='" + name + '\'' +
-               '}';
+                "name='" + name + '\'' +
+                ", types=" + types +
```

Comment:
Since this is a list you will need to make it use `Arrays.toString(types.toArray())`